docker/docker_image_dest.go

```diff
@@ -162,6 +163,7 @@ func (d *dockerImageDestination) PutBlob(ctx context.Context, stream io.Reader,
 
 	digester := digest.Canonical.Digester()
 	sizeCounter := &sizeCounter{}
+
```

Comment:
(Absolutely non-blocking: Maybe drop this? `digester` is a pretty local variable to compute `computedDigest` in this ~paragraph. But then here I am having opinions about newlines…)

Reply:
You spend more time looking at this code than I do, so of course I'll drop it.